README.rst

```diff
@@ -56,7 +56,7 @@ To install PHP-CS-Fixer, install Composer and issue the following command:
 
     $ ./composer.phar global require fabpot/php-cs-fixer
 
-Then, make sure you have ``~/.composer/vendor/bin`` in your ``PATH``, and
```

Comment:
I think these are overcomplete. Just like `Then` in this case, since there is no need for stating that you must do this at that time. Adding it to your path before is also correct since there is only a need for at the time you arive at that point. Bottomline; without a documentation style guide maybe there is no good or bad here :) Im :+1: for the change